huaweicloud/services/acceptance/cbr/resource_huaweicloud_cbr_policy_test.go

```diff
@@ -1,35 +1,43 @@
-package huaweicloud
+package cbr
 
 import (
 	"fmt"
 	"testing"
 
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
-
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/chnsz/golangsdk/openstack/cbr/v3/policies"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-
-	"github.com/chnsz/golangsdk/openstack/cbr/v3/policies"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
 )
 
 func TestAccCBRV3Policy_basic(t *testing.T) {
 	var asPolicy policies.Policy
-	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	randName := acceptance.RandomAccResourceName()
 	resourceName := "huaweicloud_cbr_policy.test"
 
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&asPolicy,
+		func(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
```

Comment:
this func need to declare a function, which is referenced by TestAccCBRV3Policy_basic and TestAccCBRV3Policy_replication.

Reply:
done